### PR TITLE
Make review setting configurable

### PR DESCRIPTION
**Pull request by alenad**

### CODE REVIEW

**Comment by alenad** [comment]
@daliwei @pierre

File: lib/paypal_express/api.rb [hunk 1]
```diff
@@ -266,7 +266,8 @@ def process_notification(notification, properties, context)
       def to_express_checkout_url(response, kb_tenant_id, options = {})
         payment_processor_account_id = options[:payment_processor_account_id] || :default
         gateway                      = lookup_gateway(payment_processor_account_id, kb_tenant_id)
-        gateway.redirect_url_for(response.token)
+        review                       = options.has_key?(:review) ? options[:review] : false
```
Comment (pierre) [inline lib/paypal_express/api.rb]: `options[:review].to_s == 'true'` to deal with blanks, etc.

Comment (alenad) [inline lib/paypal_express/api.rb]: Should I use ::Killbill::Plugin::ActiveMerchant::Utils.normalized ?

**Comment by alenad** [comment]
@pierre I updated the PR to use `::Killbill::Plugin::ActiveMerchant::Utils.normalized`

**Comment by pierre** [comment]
👍

File: lib/paypal_express/api.rb [hunk 2]
```diff
@@ -266,7 +266,7 @@ def process_notification(notification, properties, context)
       def to_express_checkout_url(response, kb_tenant_id, options = {})
         payment_processor_account_id = options[:payment_processor_account_id] || :default
         gateway                      = lookup_gateway(payment_processor_account_id, kb_tenant_id)
-        review                       = options.has_key?(:review) ? options[:review] : false
+        review                       = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(options, :review)
         gateway.redirect_url_for(response.token, :review => review)
```
Comment (pierre) [inline lib/paypal_express/api.rb]: Note: `review` might be `nil` here. Hopefully that's handled correctly upstream?

Comment (alenad) [inline lib/paypal_express/api.rb]: Yes it's handled [here](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/paypal_express_common.rb#L24)

**Pull request merged**